spec/index.html

```diff
@@ -1516,58 +1516,79 @@ <h3>The <code>rdf:JSON</code> Datatype</h3>
       <dt id="JSON-lexical-space">The <a>lexical space</a></dt>
       <dd>is the set of all <a data-lt="string">RDF strings</a> that conform to the
         <a data-cite="RFC8259#section-2">JSON Grammar</a> as described in
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]].</dd>
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of [[RFC8259]],
+        which also conform to the requirements of [[[RFC7493]]] [[RFC7493]].
+        <div class="note">
+          [[[RFC8259]]] [[RFC8259]] allows strings to include
+          <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">surrogate code points</a>
+          which are not allowed in <a data-lt="string">RDF strings</a>,
+          which are also excluded in [[RFc7493]],
+          thus the lexical representation of JSON literals excludes those including
+          <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">surrogate code points</a>.
+        </div></dd>
 
       <dt id="JSON-value-space">The <a>value space</a></dt>
-      <dd>
-        is the set of all <a data-lt="string">RDF strings</a> that conform to the
-        <a data-cite="RFC8259#section-2">JSON Grammar</a> as described in
-        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
-        [[RFC8259]], and furthermore comply with the following constraints:
+      <dd>is the set of
+        <a>strings</a>,
+        numbers (<a data-cite="XMLSCHEMA11-2#integer">integer</a> or <a data-cite="XMLSCHEMA11-2#double">double</a>),
+        <a data-cite="INFRA#ordered-map">maps</a>,
```

Comment:
```suggestion
        <a data-cite="INFRA#ordered-map">maps (unordered)</a>,
```

Reply:
It would be much better to just use something that is not ordered.   Instead you have an unordered ordered sequence, whatever that is.

Reply:
As recommended elsewhere, we use the INFRA standard for basic datatypes (other than numeric datatypes); they don't have a separate definition for an unordered map, and I don't think there is such a compatible type defined elsewhere. JSON-LD uses this for describing its [representation of a JSON object](https://www.w3.org/TR/json-ld11/#dfn-json-object).

Reply:
So an inappropriate datatype is being used just because it is in the INFRA standard?  That make no sense at all.